addons/crm/wizard/crm_lead_to_opportunity.py

```diff
@@ -42,20 +41,28 @@ def default_get(self, fields):
                 result['team_id'] = lead.team_id.id
             if not partner_id and not lead.contact_name:
                 result['action'] = 'nothing'
+            result['lead_id'] = lead.id
```

Comment:
Maybe place that right after the `lead = self.env['crm.lead'].browse(self._context['active_id'])` ?

Seems out of place.